File: frontend/Assets/Scripts/DataTransfer/Server/LeftLobbyCmd.cs
using System;
using System.IO;
using Attributes;
using Utils;


namespace DataTransfer.Server {

    [DTO (DTOCode.LeftLobby)]
    public class LeftLobbyCmd : ServerCommand {

        public static event Action <LeftLobbyCmd> OnReceived;


        public override void ReadMembers (BinaryReader reader) {}


        public override void Execute () {
            OnReceived._ (this);
        }

    }

}

File: backend/src/main/java/dto/server/LeftLobbyCmd.java
package dto.server;

import dto.DTOCode;
import dto.DTOs;
import io.netty.buffer.ByteBuf;


@DTOCode(DTOs.LEFT_LOBBY)
public class LeftLobbyCmd extends ServerCommand {

    @Override
    protected void writeMembers (ByteBuf byteBuf) {
        // nothing
    }
}
